framework/base/Security.php

```diff
@@ -30,6 +30,11 @@
  */
 class Security extends Component
 {
+    const DEV_URANDOM = '/dev/urandom';
```

Comment:
[coming in 7.1](https://wiki.php.net/rfc/class_const_visibility)

Iiuc, an undocumented class const doesn't appear in the Yii web docs, which is close enough for now. Probably even better if they are not located at the top of the class, where PhpStorm automatically put these.